master/docs/manual/customization.rst

```diff
@@ -932,18 +932,153 @@ For more information on the available commands, see :doc:`../developer/master-wo
     Step Progress
     BuildStepFailed
 
-Writing New Status Plugins
---------------------------
+Writing Dashboards with Flask_ or Bottle_
+-----------------------------------------
+
+Buildbot Nine UI is written in Javascript.
+This allows it to be reactive and real time, but comes at a price of a fair complexity.
+Sometimes, you just need a dashboard displaying your build results in your own weird manner but learning AngularJS for that is just too much.
```

Comment:
Remove "weird" and "just". Using "just" can makes things seem more trivial than the reader thinks them. It is frequently better to omit it. Using "weird" can seem overly judging. They don't think it's weird and instead may think it weird that it's so hard to do what they consider normal.